src/components/button/index.md.njk

```diff
@@ -59,10 +59,6 @@ Pages with too many calls to action make it hard for users to know what to do ne
 
 {{ example({group: "components", item: "button", example: "secondary", html: true, nunjucks: true, open: false}) }}
 
-You can use secondary buttons in combination with default buttons.
```

Comment:
I ended up moving this because it felt like the example of two buttons should really use a button group, but it seemed strange to use it before it was introduced.

If we think it's important to make this explicit at the point we're talking about secondary buttons we could bring it back, but I think we'd end up including the same example in both places?

Reply:
I totally agree.